### DISCUSSION TITLE
Nesting css not working :(

### DISCUSSION THREAD

**Discussion by projct1** [tailwindlabs/tailwindcss#11023] [Help] [answered]
I did everything exactly as described in the documentation https://tailwindcss.com/docs/using-with-preprocessors#nesting, but when building, an error is displayed:
![image](https://user-images.githubusercontent.com/9752959/232505755-0327de13-5a47-4577-90ae-ea532856a056.png)
![image](https://user-images.githubusercontent.com/9752959/232504477-3c79d0b6-5400-4486-9256-65695a9d7bad.png)
What is the problem?
Here my `postcss.config.js`:
```javascript
module.exports = {
  plugins: {
      'postcss-import': {},
      'tailwindcss/nesting': 'postcss-nesting',
      tailwindcss: {},
      autoprefixer: {},
  },
}
```
Here my `vite.config.js`:
```javascript
import { defineConfig } from 'vite'
import laravel from 'laravel-vite-plugin'
import vue from '@vitejs/plugin-vue'

export default defineConfig({
    plugins: [
        vue(),
        laravel({
            refresh: true,
            input: ['resources/css/app.css', 'resources/js/app.js']
        }),
    ],
})
```

**Comment by adamwathan** [comment, marked as answer]
Hey! You’re missing a semicolon at the end of the `@apply` statement above the nesting.

![B96E2E32-09F9-4D34-9638-B89A6765DF2A](https://user-images.githubusercontent.com/4323180/232511021-cb58f593-e002-48eb-8027-9dc9c34c80e8.png)

**Reply by projct1** [reply]
Hio! Omg, really, its true, thanks ;)
I'm used to not putting a semicolon, as unnecessary. Looks nicer:
![image](https://user-images.githubusercontent.com/9752959/232513403-3902423a-7c18-4339-bcf4-04762265be82.png)

**Reply by adamwathan** [reply]
It is necessary for any declaration other than the very last one in a rule — it works for you in that case because each rule only contains one declaration 👍 

Just like how this won't work:

```css
.foo {
  background: red
  color: yellow
}
```
